app/main.py

```diff
@@ -1,13 +1,41 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        "column": error["column_number"], "line": error["line_number"],
+        "message": error["text"], "name": error["code"], "source": "flake8"
+    }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    return {
+        "errors": [
+            {
+                "line": error["line_number"],
+                "column": error["column_number"],
+                "message": error["text"],
+                "name": error["code"],
+                "source": "flake8"
```

Comment:
Note that basically it's what format_linter_error makes. It formats in this way, so you can re-use format_linter_error function